2024/wg-fedid.html

```diff
@@ -259,20 +259,20 @@ <h3>Tentative Deliverables</h3>
         <h3>
           Other Deliverables
         </h3>
-        <p>
-          A test suite, available from <a
+        <p>Other non-normative documents will be created, such as:</p>
+        <ul>
+          <li>A test suite, available from <a
           href="https://github.com/web-platform-tests/wpt">web-platform-tests</a> as possible, must
-          be created.
-        </p>
+          be created.</li>
+        <li>A deliverable (or set of deliverables) aims to understand the threats and mitigations of Digital Credentials-related technologies concerning human rights, harm, security, and privacy. These findings will be used as input for our deliverables. This will be developed in collaboration with W3C's Technical Architecture Group (TAG), Privacy Interest Group (PING), and other relevant groups.</li>
```

Comment:
> <li>A deliverable (or set of deliverables) aims to understand the threats and mitigations of Digital Credentials-related technologies concerning human rights, harm, security, and privacy. These findings will be used as input for our deliverables. This will be developed in collaboration with W3C's Technical Architecture Group (TAG), Privacy Interest Group (PING), and other relevant groups.</li>

```suggestion
        <li>A deliverable (or set of deliverables) considering the threats and mitigations of Digital Credentials-related technologies concerning human rights, harm, security, and privacy. These findings will be used as input for our deliverables. This will be developed in collaboration with W3C's Technical Architecture Group (TAG), Privacy Interest Group (PING), and other relevant groups.</li>
```